cephalopod/factory.py

```diff
@@ -68,3 +62,17 @@ def __call__(self, *args, **kwargs):
 
     celery.Task = ContextTask
     return celery
+
+
+@multipass.identity_handler
+def identity_handler(identity_info):
+    """Implements the Flask-Multipass identity handler"""
+    cephalopod_id = identity_info.data['cephalopod_id']
```

Comment:
OK, I didn't know that was possible to change the identifier itself. So for CERN auth should we use the email or the username (e.g. `mvidalga`) as identifier? I think that currently the identifier is the username.